.github/workflows/lra-test.yml

```diff
@@ -0,0 +1,40 @@
+name: LRA test
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+  pull_request:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  lra-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Cache local Maven repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: LRA test
+      run: |
+        git config --global user.name "jbosstm"
+        git config --global user.email "jbosstm@users.noreply.github.com"
```

Comment:
@xstefank - how did you decide on this value (and the user.name above)? Is it (and the line above) required to set this for the action to work

Reply:
I believe this was required for the script. I'll try to remove it.

Reply:
yes, correct - https://github.com/xstefank/narayana/runs/2031052160

Reply:
Thanks @xstefank - I am erring on thinking that as the details are fictional, how about going with what it looks like the git output comes back with:
`git config --global user.email "you@example.com"
  git config --global user.name "Your Name"`

/cc @mmusgrov do you have any preference here?

Reply:
It's irrelevant as we are not pushing anything. This is used in the script for pulling only I believe.

Reply:
Sure, but by it not being the default (can a default really be established but the output from the command seems possible to consider as such) could it be perceived as not irrelevant and somehow meaningful?

I don't think this should block the PR as it could be changed later I suppose, but it's a question.

Reply:
Personally, I think jbosstm is better than the example the git outputs by default :) by that are my 2c. Surely, I can change it, it's not a big issue for me. I will just run it in my branch to verify it works properly.

Reply:
I tend to agree with tom (jbosstm@users.noreply.github.com implies a real email but you@example.com does not). But this job is for internal use so the change is only a "nice to have" item and need not block merging the PR.

It looks like we are all in agreement to merge the PR so provided we don't need to do anything for this last item I think we should merge and begin acquiring some data about whether the PR improves upon the rate of false positive we are seeing with our internal CI job.

Reply:
@mmusgrov please go ahead, I will open another PR when the job passes later.